pyqtgraph/Qt/__init__.py

```diff
@@ -415,11 +415,13 @@ def mkQApp(name=None):
     ============== ========================================================
     """
     global QAPP
-    
+    global PALETTE
```

Comment:
is this still in use?

Reply:
No, I went too far back when refactoring.